src/core/demangle.d

```diff
@@ -955,7 +960,7 @@ private struct Demangle
     {
         // FuncAttrs
         breakFuncAttrs:
-        while( 'N' == front )
+        while ('N' == front)
```

Comment:
While the code style is rather ugly in this file, I think we should not mix different styles.